### ISSUE TITLE
withCredentials not working correctly in options

### ISSUE THREAD

**Issue by drjaydenm** [angular-ui/ui-uploader#30]
When specifying `options.withCredentials` in the `startUpload` options object as below, it isn't respected correctly.
```javascript
$uiUploader.startUpload({
    url: 'http://my_domain.com',
    concurrency: 2,
    options: {
        withCredentials: true
    }
})
```
Looks like the uploader code is looking at the `data` key instead of the `options` key. Changing it to be `data` instead of `options` sets the XHR `withCredentials` property correctly, but this also makes the `withCredentials` value be added as a form field in the POST request.

**Comment by drjaydenm** [comment]
I have submitted a pull request #31 to fix this issue

**Comment by PowerKiKi** [comment]
Closed via #31

**Comment by drjaydenm** [comment]
Thanks, works a treat
